rest-server/src/components/auth/authController.js

```diff
@@ -0,0 +1,33 @@
+import User from '../user/userController';
```

Comment:
Destructure!! Helps with visibility. Just nice to immediately see what we're using. 

Also separate lines like the other ones